### DISCUSSION TITLE
problem on render: mitsuba-blender

### DISCUSSION THREAD

**Discussion by Sapta0802** [mitsuba-renderer/mitsuba-blender#106] [General]
I am using Blender 4.1 version (the latest version) but as an add on I have downloaded Mitsuba-blender v3.0, for that may be some problem happens for xml file export because when I try to use this xml file for sionna to preview a unwanted "cant clear json" error come.   how can i solve this. and beside Mitsuba-blender, there is a problem sign. 
![image](https://github.com/mitsuba-renderer/mitsuba-blender/assets/88283648/57143c51-4460-48bc-ad66-4e59890dda3d)

need reply urgent.

**Comment by merlinND** [comment]
Hello @Sapta0802,

You have already opened an issue in the Sionna repository about the "Can't clean for JSON" issue (https://github.com/NVlabs/sionna/discussions/452), as well as on the Colab repo (https://github.com/googlecolab/colabtools/issues/4608). It is unrelated to the `mitsuba-blender` repository, since the scene export works fine and you were able to render your scene in Mitsuba.
Please refrain from opening multiple issues about the same problem, as you are creating unnecessary work for maintainers of the various repos.

Moreover, "need reply urgent" will not help resolve your problem. Maintainers do their best to assist, but open source projects are typically provided as-is, without any guarantee of support.
